plugins/easyimage/plugin.js

```diff
@@ -94,7 +94,16 @@
 				group: 'easyimage',
 				order: 3
 			}
-		} );
+		};
+		if ( editor.plugins.link ) {
```

Comment:
The link feature is located [inside `imagebase` plugin](https://github.com/ckeditor/ckeditor-dev/blob/t/932-2961/plugins/imagebase/plugin.js#L48-L163) and this integration also should be there.